Modules/TPC/include/TPC/QualityObserver.h

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file     QualityObserver.h
+/// \author   Marcel Lesch
+///
+
+#ifndef QUALITYCONTROL_QUALITYOBSERVER_H
+#define QUALITYCONTROL_QUALITYOBSERVER_H
+
+#include "QualityControl/PostProcessingInterface.h"
+
+#include <memory>
+#include <unordered_map>
+#include <vector>
+#include <string>
+#include <TCanvas.h>
```

Comment:
Move to cxx use forward declaration here:
```suggestion
class TCanvas;
```

Reply:
Changed as requested